src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -203,6 +202,9 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
                 ))}
               </Box>
             </FormControl>
+            <FormHelperText>{errors.why?.message}</FormHelperText>
+          </Grid>
+          <Grid xs={12}>
             <Button
```

Comment:
maybe adding padding to the left would make the spacing look a little better?

Reply:
^^, you should be able to use marginleft on the button to fix it